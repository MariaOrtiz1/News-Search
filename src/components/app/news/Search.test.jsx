import React from 'react';
import { render, screen } from '@testing-library/react';
import Search from './Search';

describe('Search', () => {
  it('should displays a form with an input and submit button', () => {
    render(
      <Search
        articleTitle="bunnies"
        onArticleTitleChange={() => {}}
        onSubmit={() => {}}
      />
    );
  
    const SearchForm = screen.getByTestId('search-form');
    expect(SearchForm).toMatchSnapshot();
  });
});
