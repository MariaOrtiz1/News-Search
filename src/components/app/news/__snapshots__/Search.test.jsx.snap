// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search should displays a form with an input and submit button 1`] = `
<form
  data-testid="search-form"
>
  <label
    for="articleTitle"
  >
    Search by Keyword in Title:
  </label>
  <input
    id="articleTitle"
    type="text"
    value="bunnies"
  />
  <button
    aria-label="find-articles"
  >
    Submit
  </button>
</form>
`;
